bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/message/PowermaxInfoMessage.java

```diff
@@ -37,25 +37,28 @@ public PowermaxInfoMessage(byte[] message) {
     }
 
     @Override
-    public PowermaxState handleMessage(PowermaxCommManager commManager) {
-        super.handleMessage(commManager);
-
+    protected PowermaxState handleMessageInternal(PowermaxCommManager commManager) {
         if (commManager == null) {
             return null;
         }
 
         PowermaxState updatedState = commManager.createNewState();
 
         byte[] message = getRawData();
+        byte panelTypeNr = message[7];
+        String panelTypeStr;
 
         PowermaxPanelType panelType = null;
         try {
             panelType = PowermaxPanelType.fromCode(message[7]);
+            panelTypeStr = panelType.toString();
         } catch (IllegalArgumentException e) {
-            logger.debug("Powermax alarm binding: unknwon panel type for code {}", message[7] & 0x000000FF);
```

Comment:
Because immediately below is an unconditional call to `addDebugInfo("Panel type", panelTypeNr, panelTypeStr);` and panelTypeStr in that case will be UNKNOWN, so I didn't see the need to log the same message twice.

Reply:
Ok